app/controllers/disability_claims_base_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class DisabilityClaimsBaseController < ApplicationController
+  before_action :authorize_user
+
+  protected
+
+  def authorize_user
+    head(403) unless @current_user.evss_attrs?
```

Comment:
This logic returns a `403` for two cases:

1. Client has insufficient authorization
2. Server was unable to fetch required attributes.

A client can easily recover from case 1, but not case 2, and will need to differentiate between the two.